### PR TITLE
Add Axios Configuration

### PR DESCRIPTION
**Pull request by ldelarco**
This PR includes the basic configuration for:
- Axios: examples for GET and POST endpoints using redux sagas.

### CODE REVIEW

File: src/services/httpClient.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  client.interceptors.response.use(
+    (response) => response,
+    ({ response }) => {
+      if (!response) {
+        throw { errors: ['Connection error'] };
+      }
+
+      throw response;
+    },
+  );
```
Comment (Tintef) [inline src/services/httpClient.js:25]: Having a error doesn't mean it was a connection error 🤔 
For example, a 401 is not a connection error, same for a 403

File: src/redux/appReducer/sagas.js [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
   while (true) {
     yield take(constants.FETCH_DATA);
 
     try {
-      const data = {
-        prop1: 'example1',
-      };
+      const data = yield call(
+        exampleServices.getExample,
+      );
```
Comment (LucasBruzzone) [inline src/redux/appReducer/sagas.js:19]: Why not write all this in one line? It only has one parameter `const data = yield call(exampleServices.getExample);`

File: src/screens/Welcome/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { useDispatch } from 'react-redux';
+import {
+  View,
+  Text,
+  SafeAreaView,
+} from 'react-native';
+
+import {
+  fetchData,
+} from '../../redux/appReducer/actions';
```
Comment (LucasBruzzone) [inline src/screens/Welcome/index.js:11]: Why this in 3 lines instead of 1?

File: src/services/httpClient.js (hunk 1 above)
Comment (ldelarco) [reply src/services/httpClient.js:25]: This is only for catching when the device has no internet connection. No other errors like 401 or 403.

**Pull request merged**